### ISSUE TITLE
Review and adjust the display of search results for accessibility

### ISSUE THREAD

**Issue by Captainkirkdawson** [FreeUKGen/MyopicVicar#947]
Review against accessibility report and adjust as required
Dependant upon #945

**Comment by richpomfret** [comment]
To review/test on mobile.

**Comment by AlOneill** [comment]
The search results are not displaying as we intended on test3.

Not sure yet what the problem is exactly. Please don't implement on production until this is fixed!

**Comment by AlOneill** [comment]
Have identified a typo in show.html.erb

Browser is not getting latest freereg_content CSS file (despite several forced reloads).
However, latest freereg_content CSS is not quite correct anyway!!

That may be all -- I cannot tell until the typo and CSS are corrected.

**Comment by Captainkirkdawson** [comment]
Know what the issue is, waiting for the correct time to do the mod

**Comment by AlOneill** [comment]
Outstanding: Record details page (requires info currently in table head to be re-written as caption).
